### DISCUSSION TITLE
why the rates missing for zjets and ttbar?

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/2l2Q_limitSettingTool#65] [Comments_HIG_23_001]
Hi @jialin-guo1 ,

Do you know, why these rates not included for `zjets` and `ttbar`?

https://github.com/ram1123/2l2Q_limitSettingTool/blob/40037473abe3b74b3067660ae4e99a9af878268e/datacardClass.py#L1374-L1378

**Comment by jialin-guo1** [comment]
if you check those lines that above. you will find in each case, zjets and ttbar are actually included:
https://github.com/ram1123/2l2Q_limitSettingTool/blob/40037473abe3b74b3067660ae4e99a9af878268e/datacardClass.py#L1344-L1377

also zjet and ttbar are in workspace, see the following screen shot:
<img width="1728" alt="image" src="https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/fafdf976-722c-48bb-82df-76cb26bb8714">
